geomesa-filter/src/test/scala/org/locationtech/geomesa/filter/function/FastPropertyTest.scala

```diff
@@ -49,7 +49,7 @@ class FastPropertyTest extends Specification {
       val sf = new ScalaSimpleFeature("id", sft)
       sf.setAttributes(Array[AnyRef]("myname", "POINT(45 45)"))
 
-      val filter = ECQL.toFilter("testns:name = 'myname'", FastFilterFactory.factory)
+      val filter = FastFilterFactory.toFilter(sft, "testns:name = 'myname'")
```

Comment:
oh, so the key part in this test is that the `FastFilterFactory` was passed in to `ECQL.toFilter` - we're doing the same thing in `FastFilterFactory.toFilter`, it's just that the class has a private constructor now so you can't call it the old way (to enforce setting the thread local sft). But it is testing the same thing still.
https://github.com/elahrvivaz/geomesa/blob/7122424cb4d49343cc1d86177636048595e26fd5/geomesa-filter/src/main/scala/org/locationtech/geomesa/filter/factory/FastFilterFactory.scala#L93